README.md

```diff
@@ -2,59 +2,49 @@
 
 ## Requirements
-Deployment requires [docker-compose](https://www.docker.com/).
+Deployment requires [Docker](https://www.docker.com/).
 
 Development requires [python3](https://www.python.org/) and [nodejs](https://nodejs.org/).
 
 ## Deployment
-To deploy the website on a machine running docker run the following commands:
-```bash
-git clone https://github.com/Imageomics/Andromeda.git
-cd Andromeda/
-cp example.env .env
-docker-compose up -d
-```
-The `-d` flag runs the website in the background. For troubleshooting remove this flag to more easily monitor the deployment.
+Our primary deployment environment is [Hugging Face Spaces](https://huggingface.co/spaces/imageomics/Andromeda).
 
-Once the website finishes launching the website will be available on port 80: [http://localhost](http://localhost).
-Part of the deployment builds the docker containers used by the website, so it may take a few minutes to finish deployment.
-To stop the app run `docker-compose down`.
+To build a Docker image from the source code and deploy the container:
 
-Settings for the app can be changed in `.env`.
-Options for `.env`:
-- BACKEND_WORKERS - number of [workers used by gunicorn](https://docs.gunicorn.org/en/latest/run.html#commonly-used-arguments)
-- ANDROMEDA_RGB_SATELLITE_URL - URL pointing to a CSV file for joining RGB data
+1. Clone this repository and navigate into it.
 
-### Certbot Setup on AWS
-Certificates can be installed to the EC2 VM using Certbot with Nginx on pip following [EFF instructions](https://certbot.eff.org/instructions?ws=nginx&os=pip).
+2. Build the Docker image.
 ```bash
-sudo python3 -m venv /opt/certbot/
-sudo /opt/certbot/bin/pip install --upgrade pip
-sudo /opt/certbot/bin/pip install certbot certbot-nginx
-sudo ln -s /opt/certbot/bin/certbot /usr/bin/certbot
+docker image build -t andromeda .
 ```
-And to install Nginx on the Amazon Linux 2023 (following [instructions](https://awswithatiq.com/how-to-install-nginx-in-amazon-linux-2023/)) and run Certbot:
+
+3. Optionally, set your environment variables in `.env`. See the section below for more information.
 ```bash
-docker-compose down
-sudo dnf update -y
-sudo dnf install nginx -y
-sudo systemctl start nginx
-sudo certbot certonly --nginx
+cp example.env .env # uses the provided example environment variables
 ```
-Following prompted agreements, registration, and specifying andromeda.imageomics.org, the certificates are installed to:
-
-Certificate: `/etc/letsencrypt/live/andromeda.imageomics.org/fullchain.pem`
-
-Key: `/etc/letsencrypt/live/andromeda.imageomics.org/privkey.pem`
 
-Finally, shut down Nginx on the system to avoid interference with the container:
+4. Run the Docker container.
 ```bash
-sudo systemctl stop nginx
-docker-compose up -d
+docker container run --env-file .env -p 7860:7860 andromeda
 ```
 
+5. Prepare a simple Dockerfile in a Hugging Face Space. For example:
+```Dockerfile
+FROM ghcr.io/Imageomics/Andromeda:latest
```

Comment:
```suggestion
FROM ghcr.io/Imageomics/Andromeda:latest
COPY run.sh /api/run.sh
```
With the `run.sh` file in place, this should be the way to set up the HF Dockerfile.

Reply:
That file is located at `andromeda/run.sh` and copied into the container with `COPY ./andromeda/. /api`.

On the HF Dockerfile, we should only need to retrieve the image from GHCR.